Dockerfile.dev.front

```diff
@@ -4,7 +4,8 @@ LABEL "repository"="https://github.com/uqmc/arapiles"
 LABEL "homepage"="https://github.com/uqmc/arapiles"
 
 WORKDIR /front
-RUN rm -rf ./node_modules/
+COPY src/front/package.json .
```

Comment:
When dealing with NPM, you should copy over the `package-lock.json` file as well so that the versioning of dependencies is consistent. When the lock file is present, NPM will install from it instead of `package.json`.

Replace with `COPY src/front/package*.json .`

Reply:
No worries, should be fixed now.